cli.js

```diff
@@ -31,6 +34,13 @@ function run() {
       process.cwd(),
     ], {stdio: 'inherit'});
     break;
+  case 'link':
+    if (!args[1]) {
+      console.error('Usage: react-native link <path>');
```

Comment:
@tadeuzagallo - good idea, the ideas are certainly not mutually exclusive - if we later add support for @frantic's proposed api (which looks ideal certainly) we can just switch between the two depending on the arity. I can imagine a number of edges cases that would get in the way of `react-native link react-native-video`, which would require more careful consideration, and it's not a big hassle to include in the README for a project `npm i x --save && react-native link MyProject node_modules/x/X`